src/app/dashboard/layout.tsx

```
@@ -8,8 +8,9 @@ import { usePrefetchData } from "@/components/modules/auth/hooks/usePrefetchData
 import { PayoutContextProvider } from "@/components/modules/payouts/context/PayoutContext";
 import { RoleSelectionProvider } from "@/components/providers/role-selection.provider";
 import { SidebarInset, SidebarProvider } from "@/components/ui/sidebar";
+import { ReactNode } from "react";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)